app/admin/traits/Assignable.php

```diff
@@ -74,14 +76,14 @@ public function updateAssignTo($group = null, $assignee = null)
         if (!is_null($assignee))
             $this->assignee()->associate($assignee);
 
-        $this->fireSystemEvent('admin.assignable.beforeAssignTo', [$group, $assignee, $oldAssignee]);
+        event(new BeforeAssignTo($this, $group, $assignee, $oldAssignee));
```

Comment:
... we fire both 
```
$this->fireSystemEvent('admin.assignable.beforeAssignTo', [$group, $assignee, $oldAssignee]);
        event(new BeforeAssignTo($this, $group, $assignee, $oldAssignee));
```

This way we can come back and remove the legacy event before v5